discriminable.gemspec

```diff
@@ -41,5 +41,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "minitest", "~> 5.15"
   spec.add_development_dependency "rubocop", "~> 1.26"
   spec.add_development_dependency "simplecov", "~> 0.21"
+  spec.add_development_dependency "simplecov_json_formatter", "~> 0.1"
```

Comment:
This line shouldn't be necessary, `simplecov_json_formatter` is one of `simplecov`'s `0.21.0` dependencies.

Reply:
Thanks so much for the feedback. I have removed it again.